kotlin-native/backend.native/tests/debugger/src/test/kotlin/org/jetbrains/kotlin/native/test/debugger/LldbTests.kt

```diff
@@ -495,4 +495,44 @@ class LldbTests {
         """.trimIndent().lldb(program)
     }
 
+    @Test
+    fun `inline function parameters are visible`() = lldbTest("""
+        data class A(var x: Int)
+
+        inline fun foo(p1: Int, p2: Int, f: (A, Int, Int) -> Unit) { 
+            val tmp = A(0)
+            println()
+            f(tmp, p1, p2)
+        }
+
+        fun main() {
+            val tmp1 = 1
+            val tmp2 = 2
+            foo(tmp1, tmp2) { pix, piy, piz ->
```

Comment:
It seams, that the current code works only if the variable is passed to an argument. 

For example, if I change the code to `foo(1, 2)` variables disappear. Also p2 disappear if use `foo(tmp1, tmp1+tmp1)`. Can you please check it, and add more complex tests?